.github/workflows/aws_openshift_rosa_hcp_single_region_tests.yml

```diff
@@ -0,0 +1,646 @@
+
+            - name: 🧪 TESTS - Run additional tests
+              if: env.TESTS_ENABLED == 'true'
+              timeout-minutes: 20
+              run: |
+                  set -euxo pipefail # tolerate, nothing.
+
+                  echo "Show zeebe cluster topology using generic/kubernetes/single-region/procedure/check-zeebe-cluster-topology.sh:"
+                  export ZEEBE_CLIENT_ID="${{ steps.secrets.outputs.CI_CAMUNDA_USER_TEST_CLIENT_ID }}"
+                  export ZEEBE_CLIENT_SECRET="${{ steps.secrets.outputs.CI_CAMUNDA_USER_TEST_CLIENT_SECRET }}"
+                  export DOMAIN_NAME="$CAMUNDA_DOMAIN"
+
+                  # Execute the script and capture the output in a variable
+                  check_zeebe_topology_output=$(./generic/kubernetes/single-region/procedure/check-zeebe-cluster-topology.sh)
+
+                  echo "$check_zeebe_topology_output" | jq
+
+                  # Checks
+                  error_found=false
+                  check_zeebe_topology_all_healthy=$(echo "$check_zeebe_topology_output" | jq '[.brokers[].partitions[].health == "healthy"] | all')
+                  check_zeebe_topology_cluster_size=$(echo "$check_zeebe_topology_output" | jq '.clusterSize')
+                  check_zeebe_topology_partitions_count=$(echo "$check_zeebe_topology_output" | jq '.partitionsCount')
+
+                  if [ "$check_zeebe_topology_all_healthy" = "true" ]; then
+                    echo "✅ All partitions are healthy."
+                  else
+                    echo "❌ Not all partitions are healthy"
+                    error_found=true
+                  fi
+
+                  if [ "$check_zeebe_topology_cluster_size" -eq 3 ]; then
+                    echo "✅ Cluster size is 3."
+                  else
+                    echo "❌ Cluster size is not 3."
+                    error_found=true
+                  fi
+
+                  if [ "$check_zeebe_topology_partitions_count" -eq 3 ]; then
+                    echo "✅ Partitions count is 3."
+                  else
+                    echo "❌ Partitions count is not 3."
+                    error_found=true
+                  fi
+
+                  echo "Comparing golden file of the zeebe topology output..."
+
+                  reference_file="./generic/kubernetes/single-region/procedure/check-zeebe-cluster-topology-output.json"
+                  # Save the output to a temporary file
+                  temp_output=$(mktemp)
+                  echo "$check_zeebe_topology_output" > "$temp_output"
+
+                  # Order each file also remove not predictable fields
+                  yq e '.brokers |= sort_by(.host) | .brokers[] |= (.partitions |= sort_by(.partitionId) | .partitions[].role = "NOT_PREDICTABLE")' -i "$temp_output"
+                  yq e '.brokers |= sort_by(.host) | .brokers[] |= (.partitions |= sort_by(.partitionId) | .partitions[].role = "NOT_PREDICTABLE")' -i "$reference_file"
```

Comment:
wouldn't you also have to remove the version, likely to clash.

Reply:
I guess it's fine to keep it but definitely a 100% error source once a month after each patch release.